app/models/model_1.rb

```diff
@@ -0,0 +1,6 @@
+class Model1 < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, stretches: 13
```

Comment:
Kindly make sure that devise settings is implemented inside user.rb model not here 👍